cli/src/funTest/kotlin/OrtMainFunTest.kt

```diff
@@ -63,7 +63,7 @@ class OrtMainFunTest : StringSpec() {
                     packageCurationProviders = listOf(
                         ProviderPluginConfiguration(
                             type = "File",
-                            config = mapOf("path" to getAssetFile("gradle-curations.yml").path)
+                            options = mapOf("path" to getAssetFile("gradle-curations.yml").path)
```

Comment:
Also here, I believe a better / stronger rationale for this change would be that we need to distinguish from `secrets`, which otherwise could also be seen as part of the generic "config".